### DISCUSSION TITLE
YimMenu is moving!

### DISCUSSION THREAD

**Discussion by Yimura** [YimMenu/YimMenu#266] [Announcements]
In the coming weeks I will move YimMenu to an organisation.

With an organisation we can better maintain YimMenu and allow for more people to contribute easily.

**Comment by Yimura** [comment]
## DO NOT DO THIS WHEN YOU HAVE A FORK
People having problems with git you can override the repository URL manually using either git commands or editing a file.

### How to update
#### Git Command
```bash
git remote set-url origin https://github.com/YimMenu/YimMenu.git
```

#### File Edit

`.git/config`:
```ini
[remote "origin"]
	url = git@github.com:YimMenu/YimMenu.git # replace with "https://github.com/YimMenu/YimMenu.git"
	fetch = +refs/heads/*:refs/remotes/origin/*
```

### SSH URL

If you prefer to use the SSH url here it is:
```
git@github.com:YimMenu/YimMenu.git
```

**Comment by maybegreat48** [comment]
Anyone know why the decompiled scripts repo from @sainan was moved to YimMenu?

**Reply by Yimura** [reply]
More like @Sainan took them down and I copied it over to the @YimMenu organisation.

At which point I took the liberty to add some tags which help to go back to exact versions of the scripts.
Personally I have never bothered figuring out how to decompile scripts because I could just use the one's from Sainan.

**Reply by Sainan** [reply]
Then get to learning, skid.
